playbooks/roles/common/tasks/main.yml

```diff
@@ -181,14 +181,6 @@
   become: True
   when: config_templates.changed
 
-- name: Add ntp alert script
-  template:
-    src: "log-ntp-alerts.sh.j2"
-    dest: "{{ COMMON_BIN_DIR }}/log-ntp-alerts.sh"
-    owner: root
-    group: root
-    mode: "0755"
```

Comment:
I commented on the ticket about whether we want to find a way to upstream this stuff (basically through introducing variables and `when` statements in the right places) or not, or roll our own playbook. If we wanted to upstream, then deleting this wouldn't work out, and we'd have to gate it with a `when` + a variable.

Reply:
Seeing how much trouble this seems to be, I suggest to simply keep ntpd.  There isn't much to be gained by removing it, so let's focus on the services that use more memory and are easier to get rid of.